sunpy/coordinates/great_arc.py

```diff
@@ -0,0 +1,121 @@
+#
+# Calculates the co-ordinates along great arcs between two specified points
+# which are assumed to be on disk.
+#
+from __future__ import absolute_import, division, print_function
+
+import numpy as np
+from astropy.coordinates import SkyCoord
+import sunpy.coordinates
+
+
+def great_arc(start_point, end_point, center=None, number_points=100):
+    """
+    Calculate a user-specified number of points on a great arc between a start
```

Comment:
suggested change `Calculates a user-specified number of points on an arc of a great circle connecting two points`